harness/tests/experiment/utils.py

```diff
@@ -174,6 +177,61 @@ def repo_path(path: str) -> str:
     return os.path.join(os.path.dirname(__file__), "../../../", path)
 
 
+def e2e_fixtures_path(path: str) -> str:
+    # This function is a helper for testing, all necessary fixtures should be refactored instead.
+    # TODO: refactor fixtures.
+    return os.path.join(os.path.dirname(__file__), "../../../e2e_tests/tests/fixtures", path)
+
+
+def cv_examples_path(path: str) -> str:
+    return os.path.join(os.path.dirname(__file__), "../../../examples/computer_vision", path)
+
+
+def gan_examples_path(path: str) -> str:
+    return os.path.join(os.path.dirname(__file__), "../../../examples/gan", path)
+
+
+def tutorials_path(path: str) -> str:
+    return os.path.join(os.path.dirname(__file__), "../../../examples/tutorials", path)
+
+
+def import_module(module_name: str, module_path: str, model_context: Optional[str] = None) -> Any:
```

Comment:
why not use the shiny new `det.import_from_path` that I wrote?

This is basically what it's designed for.